source/server/server_configuration_parser.cpp

```diff
@@ -161,7 +162,41 @@ FeatureToggles ServerConfigurationParser::parse_feature_toggles() const
       throw InvalidFeatureToggleException(ex.what());
     }
   }
-  return FeatureToggles(std::move(map), FeatureToggles::CodeReadiness::kRelease);
+  return FeatureToggles(std::move(map), parse_code_readiness());
+}
+
+FeatureToggles::CodeReadiness ServerConfigurationParser::parse_code_readiness() const
+{
+  auto code_readiness_it = config_file_.find(kCodeReadinessKey);
+  if (code_readiness_it != config_file_.end()) {
+    try {
+      auto readiness_token_value = code_readiness_it->get<std::string>();
+      std::transform(
+          readiness_token_value.begin(),
```

Comment:
Note: would it be better to also erase `_`s? Or perhaps just to accept both `nextrelease` and `next_release`? I like for it to be easier to remember what to enter for these tokens.